Examples/Tests/initial_distribution/analysis_distribution.py

```diff
@@ -261,6 +261,34 @@
 assert(f7_error < tolerance)
 
 
+#============================================
+# Gaussian with parser mean and standard deviation
+#============================================
+
+# load data
+bin_value_ux, bin_data_ux = read_reduced_diags_histogram("h9x.txt")[2:]
+bin_value_uy, bin_data_uy = read_reduced_diags_histogram("h9y.txt")[2:]
+bin_value_uz, bin_data_uz = read_reduced_diags_histogram("h9z.txt")[2:]
+
+def Gaussian(mean, sigma, u):
+    V = 8.0 # volume in m^3
+    n = 1.0e21 # number density in 1/m^3
```

Comment:
Could these be parsed from the input file using `Tools/Parser/input_file_parser.py?`